src/containers/setContacts/ducks/actions.ts

```diff
@@ -0,0 +1,15 @@
+import { genericAsyncActions } from '../../../utils/asyncRequest';
+import { ContactInfo } from './types';
+
+export const contacts = genericAsyncActions<ContactInfo, any>();
+export const setContacts = genericAsyncActions<void, any>();
```

Comment:
Does the setContacts API return the ContactInfo in the response? Either way, I think we can get away with one action, probably.